bundles/org.openhab.binding.heliosventiremote/src/main/java/org/openhab/binding/heliosventiremote/internal/HeliosVentiRemoteConfiguration.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.heliosventiremote.internal;
+
+/**
+ * The {@link HeliosVentiRemoteConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Raphael Mack - Initial contribution
+ */
+public class HeliosVentiRemoteConfiguration {
+
+    public static final String SERIAL_PORT = "serialPort";
+    public static final String POLL_PERIOD = "pollPeriod";
+
+    /**
+     * Port name for a serial connection to RS485 bus. Valid values are e.g. COM1 for Windows and /dev/ttyS0 or
+     * /dev/ttyUSB0 for Linux.
+     */
+    public String serialPort;
+
+    /**
+     * The Panel Poll Period. Default is 60 sec. = 1 minute;
+     */
+    public Integer pollPeriod;
```

Comment:
yes, but I would have duplicated the magic number 60 in the xml and in java. Is that good practice? Personally I like having things in only one place.

Reply:
Currently we have that at a lot of places. Maybe it should be discussed more general than here. At least it simplifies the code if you add it here.

Reply:
in this case of the defaults for settings it might not be too critical even if the worst case happened and they would diverge. That is not mission critical for the binding and OH in general.